### PR TITLE
Add coverage stuffs

### PR DESCRIPTION
**Pull request by rmevans9**
This does not hit the goal of 100% coverage because we are not there and that would cause a world of pain (well... if we were to hook it up to prepush hooks or anything of that nature). This introduces generating coverage per request

Fixes: #62

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -69,6 +69,7 @@
     "mockery": "2.1.0",
     "npm-run-all": "4.0.2",
     "npm-scripts-info": "0.3.6",
+    "nyc": "^11.1.0",
```
Comment (skellock) [inline package.json]: Can we lock this bad boy down?

**Comment by skellock** [comment]
Thanks Rich!  What did we score?  like 50%?

File: package.json (hunk 1 above)
Comment (rmevans9) [inline package.json]: ahhhhhhhh. Fix landing soon

**Comment by rmevans9** [comment]
screen shot 2017-08-13 at 9 19 51 pm

**Pull request merged**